groups/cvo/variables.tf

```diff
@@ -194,6 +194,11 @@ variable "cifs_client_cidrs" {
   default     = []
 }
 
+variable "peering_cidrs" { 
+  type        = list(any) 
+  description = "The full CIDR formatted IP used for cluster peering from Cardiff nodes to all AWS CVO instances " 
```

Comment:
The variable is defined as a list and the description should reflect this. May also be worth removing any location references since the variable name is generic, the use of the list can be more generic too.

The variable type can be further defined as `list(string)`

Reply:
Completed as required